csharp/Microsoft.Azure.Databricks.Client/Run.cs

```diff
@@ -100,5 +100,29 @@ public class Run : RunIdentifier
         /// </summary>
         [JsonProperty(PropertyName = "run_page_url")]
         public string RunPageUrl { get; set; }
+
+        /// <summary>
+        /// The time at which this run was finished in epoch milliseconds (milliseconds since 1/1/1970 UTC).
+        /// </summary>
+        [JsonProperty(PropertyName = "start_time")]
```

Comment:
Should be [JsonIgnore] here. The EndTime is not transferred from/to the REST API, so no serialization/deserialization should happen. Also the next line [JsonConverter(...)] should be removed.

Reply:
Here is [an example](https://www.newtonsoft.com/json/help/html/PropertyJsonIgnore.htm) how to use JsonIgnore.

Reply:
Ha yes, good point - I was too quick!